Gemfile.lock

```diff
@@ -42,7 +47,7 @@ GEM
       rspec (>= 2.99.0, < 4.0)
     hashdiff (0.3.2)
     honeybadger (3.1.0)
-    json (2.0.3)
+    json (2.1.0)
```

Comment:
Nitpick: I would prefer to not upgrade runtime dependencies in a PR that deals with CI.